pkg/pod/pod.go

```diff
@@ -77,6 +79,14 @@ var (
 	}}
 )
 
+func getRunner(k kubernetes.Interface) string {
+	cfg, err := configmap.Load("config-entrypoint")
+	if err != nil {
+		return ""
+	}
+	return cfg["runner"]
+}
```

Comment:
this seems a bit different from how we interact with config maps in other parts of the code, i think for other config maps we use https://github.com/tektoncd/pipeline/blob/master/pkg/apis/config/default.go